ui/src/app/job-details/tabs/tabs.component.spec.ts

```diff
@@ -190,7 +220,7 @@ describe('JobTabsComponent', () => {
     let de: DebugElement = fixture.debugElement;
 
     expect(de.queryAll(By.css('mat-expansion-panel.list-row div.task-name')).length)
-      .toEqual(tasks.length);
+      .toEqual(tasks.length + attempts.length);
```

Comment:
This seems like an unexpectedly substantive change. Was there a bug here?

Reply:
Yeah, I think Old Angular reliably ran these tests in the order they appear in the file, while New Angular runs them in a random order. A test further down in this file modified a member of `tasks` by adding two attempt objects to it, which results in this DOM query returning two additional rows (iff that test runs first). I "fixed" the broken test by adding the two attempts to the task at creation time, so the test fixture appears the same to every test in the file no matter when its run.